### PR TITLE
MacOS: rename to Red Hat OpenShift Local

### PR DESCRIPTION
**Pull request by gbraad**

### CODE REVIEW

**Comment by gbraad** [comment]
@anjannath please take a look

File: Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ PODMAN_VERSION ?= 3.4.4
 BUNDLE_EXTENSION = crcbundle
 CRC_VERSION = 2.1.0
 COMMIT_SHA=$(shell git rev-parse --short HEAD)
-MACOS_INSTALL_PATH = /Applications/CodeReady Containers.app/Contents/Resources
+MACOS_INSTALL_PATH = /Applications/Red Hat OpenShift Local.app/Contents/Resources
```
Comment (anjannath) [inline Makefile:8]: also need to update the directory path in [L#316](https://github.com/code-ready/crc/blob/609ecb246881d17325048e2b21301f7a1522122d/Makefile#L316)

File: packaging/darwin/Distribution.in [hunk 2]
```diff
@@ -21,22 +21,22 @@ function installCheck() {
 	var apps = system.applications.fromIdentifier('com.redhat.codeready.containers');
 	if(apps) {
         my.result.title = 'Update failed';
-        my.result.message = 'CodeReady Containers is running. Please stop the application before updating.';
+        my.result.message = 'Red Hat OpenShift Local is running. Please stop the application before updating.';
```
Comment (anjannath) [inline packaging/darwin/Distribution.in:24]: Same file L#3 also needs to be updated, contains `<title>CodeReady Containers</title>`

File: Makefile (hunk 1 above)
Comment (gbraad) [reply Makefile:8]: different format in those lines due to the spaces.

**Comment by praveenkumar** [comment]
FYI, when we have older version installed and install this one then in the Applications folder both the app exist. we need to tell users that remove older version manually ( rm -fr /Applications/CodeReady\ Containers.app)

**Pull request merged**